include/wx/msw/ole/oleutils.h

```diff
@@ -66,18 +66,45 @@ inline void wxOleUninitialize()
 class WXDLLIMPEXP_CORE wxBasicString
 {
 public:
-    // ctors & dtor
-    wxBasicString(const wxString& str);
-    wxBasicString(const wxBasicString& bstr);
-    ~wxBasicString();
 
-    wxBasicString& operator=(const wxBasicString& bstr);
-
-    // accessors
-        // just get the string
+	// Takes over the ownership of bstr
+    wxBasicString(BSTR bstr = NULL);
```

Comment:
Typical example is a function taking (const reference to) `wxBasicString`. It could be surprising to "lose" `BSTR` by passing it to such function accidentally (because the compiler would create a temporary `wxBasicString` that would take ownership of it implicitly, if the ctor is not explicit). I think the `Get()`/memory leaks debacle shows that we really should prefer explicitness.

As for `bool copy`, I really don't like it. If we want to have a choice here (but I don't think we do), we should rather have `static wxBasicString TakeOwnership(BSTR)` and `static wxBasicString MakeCopy(BSTR)` functions instead.